content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,65 @@
+
+{{< image width="100%"
+    link="single-cluster-external-Istiod.svg"
+    alt="Istio mesh in a single cluster with Istiod outside"
+    title="Single cluster Istio mesh with Istiod outside"
+    caption="Single cluster Istio mesh with Istiod in an external control plane cluster"
+    >}}
+
+Mesh admins can expand the service mesh to multiple clusters, which are managed by the same Istiod running in the `external control plane cluster`. The `config cluster` also serves as the Istio configuration cluster, in addition to run services. The `external control plane cluster` reads Istio configurations from the `primary data plane cluster` and pushes them to both the `config cluster` and `remote cluster` per the diagram below.
+
+{{< image width="100%"
+    link="multiple-clusters-external-Istiod.svg"
+    title="Multicluster Istio mesh with Istiod outside"
+    caption="Multicluster Istio mesh with Istiod in an external control plane cluster"
+    >}}
+
+Mesh operators may further expand this deployment model to manage multiple Istio control planes from an external control plane cluster that runs multiple Istiod control planes:
+
+{{< image width="100%"
+    link="multiple-external-Istiods.svg"
+    alt="Istio meshes in single clusters with Istiod outside"
+    title="Multiple single clusters Istio meshes with Istiod outside"
+    caption="Multiple single clusters with multiple Istiod control planes in an external control plane cluster"
+    >}}
+
+The `external control plane cluster` can be used to host multiple Istiod control planes and each Istiod manages its own `config cluster`. Mesh operators could install another Istio mesh in the `external control plane cluster` and configure its `istio-ingress` gateway to route traffic from a config cluster to its corresponding Istiod control plane.
+
+## Conclusion
+
+This new deployment model enables the Istio control plane to be run and managed by mesh operators who have operational expertise in Istio. Mesh operators can run the control plane in their own control plane clusters or other environments, provide it as a service to mesh admins. Mesh operators can optionally run multiple Istiod control planes in the control plane cluster, deploy their own Istio mesh and use their `istio-ingress` gateway to control access to these Istiod control planes. Through these examples provided here, mesh operators can explore different implementation choices and choose what works best for them.
```

Comment:
Consider .... "This new deployment model enables the Istio control plane to be run and managed by mesh operators who have operational expertise and providing a clean separation between service mesh control and data planes."

Reply:
Updated, thanks Dan!